ArchiSteamFarm/IPC/Startup.cs

```diff
@@ -130,8 +133,47 @@ internal sealed class Startup {
 
 			// The order of dependency injection matters, pay attention to it
 
+			// Add support for custom reverse proxy endpoints
+			HashSet<string>? knownNetworksText = Configuration.GetSection("Kestrel:KnownNetworks").Get<HashSet<string>>();
+
+			HashSet<IPNetwork>? knownNetworks = null;
+
+			if (knownNetworksText != null) {
+				knownNetworks = new HashSet<IPNetwork>(knownNetworksText.Count);
+
+				foreach (string ipAddressText in knownNetworksText) {
+					string[] addressParts = ipAddressText.Split('/');
```

Comment:
Add `RemoveEmptyEntries` to split options, totally unneeded in this case, but will avoid additional CPU cycles if somebody uses `IP/`.